src/components/JoinButton.jsx

```diff
@@ -0,0 +1,25 @@
+const JoinButton = () => {
+  // Inline style for the button
+  const buttonStyle = {
+    backgroundColor: "#f0c14b",
+    color: "black",
+    padding: "10px 20px",
+    border: "none",
+    borderRadius: "5px",
+    cursor: "pointer",
+    textDecoration: "none",
+  };
+
+  return (
+    <a
```

Comment:
Use Next Link instead of a tags 

https://acm-ucr.github.io/docusaurus/docs/nextjs/next-link